app/components/check_answers_summary_list_card_component.rb

```diff
@@ -15,4 +15,26 @@ def applicable_questions
   def get_answer_label(question)
     question.answer_label(log).presence || "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
   end
+
+  def check_answers_card_title(question)
+    if question.form.type == "lettings"
+      case question.check_answers_card_number
+      when 1
+        "Lead tenant"
+      when 2..8
+        "Person #{question.check_answers_card_number}"
+      end
+    elsif question.form.type == "sales"
+      case question.check_answers_card_number
+      when 1..number_of_buyers
+          "Buyer #{question.check_answers_card_number}"
+      when (number_of_buyers + 1)..(number_of_buyers + 4)
+          "Person #{question.check_answers_card_number - number_of_buyers}"
```

Comment:
using `person_display_name` caused issues for the `person_known` questions as the `person_index` follows a different pattern (1-4) rather than 2-5 or 3-6 depending on jointpur. This was the simplest solution to that